src/query/src/sql.rs

```diff
@@ -501,6 +501,51 @@ pub async fn show_index(
     .await
 }
 
```

Comment:
Please add comments to this public function.

Reply:
Resolved in [d5b0776](https://github.com/GreptimeTeam/greptimedb/pull/5782/commits/d5b07767973c6f18fa906addc27e2021f493ae34)